islandora_internet_archive_bookreader.module

```diff
@@ -43,6 +43,14 @@ function islandora_internet_archive_bookreader_menu() {
       'file' => 'includes/callback.inc',
       'type' => MENU_CALLBACK,
     ),
+    'internet_archive_bookreader_get_image_uri/%islandora_object' => array(
+      'page callback' => 'islandora_internet_archive_bookreader_get_image_uri_callback',
+      'page arguments' => array(1),
+      'access callback' => 'user_access',
```

Comment:
`user_access()` is the default, though I suspect this should be `islandora_object_access()`.